algorithmic_efficiency/spec.py

```diff
@@ -238,5 +280,5 @@ def data_selection(workload: Workload, input_queue: Iterator[Tuple[Tensor,
 
 
 def get_batch_size(workload_name):
-  """Return a batch size to use for a given workload."""
+  """Return a per-device batch size to use for a given workload."""
```

Comment:
I think you now decided that `get_batch_size` should return the global batch size instead of the per-device one, no?

Reply:
yep, this was out of sync with the GNN workload PR, I merged and updated.